README.md

```diff
@@ -82,12 +109,22 @@ cd infrastructure/chroma_server_k8s
 kubectl apply -f .
 ```
 
-Run the embedding pipeline.
+Port-forward the chroma server service to localhost using the following command. This will ensure we can access the server from localhost.
+
+```bash
+kubectl port-forward service/server 8000:8000
+```
+
+Finally, in a separate terminal we can run the data embedding pipeline.
 
 ```bash
-python run.py -e
+python run.py --embed
 ```
 
+In data embedding pipeline, we take the validated dataset from data preparation pipeline and use Chroma vector database to store the embedding of the text data. This pipelines uses both the Mind and NHS data.
```

Comment:
I'd put this above the bash command, sets the scene for the user prior to them running it - they need to know what's going to happen.